bundles/org.eclipse.sirius.emfjson/src/main/java/org/eclipse/sirius/emfjson/resource/JsonResource.java

```diff
@@ -425,6 +428,12 @@ interface IEObjectHandler {
      */
     Object OPTION_SAVE_FEATURES_ORDER_COMPARATOR = "OPTION_SAVE_FEATURES_ORDER_COMPARATOR"; //$NON-NLS-1$
 
+    /**
+     * Specify a {@link Function} returning {@link Boolean#TRUE} if a given {@link EDataType} should be serialized
```

Comment:
good remark from Gionatta. Prefer a Predicate that is designed for this need.

Reply:
Definitely yes! Thank you for the remark. I fixed that.